hci/command/commands/hci_ext_packet_error_rate_cmd.py

```diff
@@ -0,0 +1,44 @@
+from struct import pack, unpack
+
+from ..command_packet import CommandPacket
+from enum import IntEnum
+
+
+class HCI_EXT_PacketErrorRateCmd(CommandPacket):
+
+    class Command(IntEnum):
+        HCI_EXT_PER_RESET = 0x00
+        HCI_EXT_PER_READ = 0x01
+
+    def __init__(self, conn_handle, command):
+        super().__init__(
+            CommandPacket.OpCode.HCI_EX_PER,
```

Comment:
This OpCode does not exist. Please, test your command before sending a pull request. You can test it the same way we show on our README.md example.